### PR TITLE
Revise staging guide

### LINKED ISSUE #114: add note about not submitting out of /staging
Add to file-availability-largedata in the policies section. There's a note in the submit file section but it would be good to make more visible.

### PR DESCRIPTION
**Pull request by jmvera255**
revisions to staging guide include a quickstart "recipe" of instructions, collapsible sections to prevent information overload, and incorporation of file transfer graphics
closes #114

### CODE REVIEW

File: file-avail-largedata.md [hunk 1]
```diff
@@ -1,299 +1,603 @@
...
+directory on your submit server. 
+
+CHTC's `/staging` location is specifically intended for:
+
+- any individual input files >100MB   
+- input files totaling >500MB per job
+- individual output files >4GB
+- output files totaling >4GB per job 
+
+# Table of Content
```
Comment (ChristinaLK) [inline file-avail-largedata.md:35]:
```suggestion
# Table of Contents
```

File: file-avail-largedata.md [hunk 2]
```diff
@@ -1,299 +1,603 @@
...
+- [Use The Transfer Server To Move Files To/From Staging](#transfer)       
+- [Submit Jobs With Input Files in Staging](#input)     
+- [Submit Jobs That Transfer Output Files To Staging](#output)
+- [Tips For Success When Using Staging](#tips)
+- [Managing Staging Data and Quotas](#quota)    
+
+<a name="use"></a>
+# Who Should Use `/staging`
+
+`/staging` is a location specifically for hosting singularly larger input (>100MB)
```
Comment (ChristinaLK) [inline file-avail-largedata.md:50]: I'm not sure about "singularly" here, but don't have any good suggestions. "Individual" maybe?

File: file-avail-largedata.md [hunk 3]
```diff
@@ -1,299 +1,603 @@
...
+<code>error</code>, and <code>output</code> files should never be 
+written to <code>/staging</code>.</p>
+
+- **Backup your files**: As with all CHTC file spaces, CHTC does not back 
+up your files in `/staging`.
+
+- **Use bash script commands to access files in `/staging`**: Files placed in `/staging` 
+should **NEVER** be listed in the submit file, but rather accessed 
+via the job's executable (aka .sh) script. More details provided 
+[below](TBD).
```
Comment (ChristinaLK) [inline file-avail-largedata.md:85]: #input ?

File: file-avail-largedata.md [hunk 4]
```diff
@@ -1,299 +1,603 @@
...
+transfer server, transfer.chtc.wisc.edu, instead of the submit server,
+to upload and download appropriate files to and from `/staging`. Transferring 
+files to `/staging` with the submit server can negatively impact job submission for 
+you and other users. More details provide [below](#transfer).
+
+- **Quota control**:`/staging` directories include disk space and 
+items (i.e. directories and files) quotas. Quotas are necessary for 
+maintaning the stability and reliability of `/staging`. Quota changes can 
+be requested by emailing <chtc@cs.wisc.edu> and 
+users can monitor quota settings and usage as described [below](TBD)
```
Comment (ChristinaLK) [inline file-avail-largedata.md:97]:
```suggestion
users can monitor quota settings and usage as described [below](#quota)
```

File: file-avail-largedata.md [hunk 5]
```diff
@@ -1,299 +1,603 @@
...
+
+- **Reduce file size and count**: We expect that users will use `tar` and 
+compression to reduce data size and file counts such that a single tarball 
+is needed and/or produced per job. More details provided [below](TBD).
```
Comment (ChristinaLK) [inline file-avail-largedata.md:101]:
```suggestion
is needed and/or produced per job. More details provided [below](#tips).
```

File: file-avail-largedata.md [hunk 6]
```diff
@@ -1,299 +1,603 @@
...
 
-> **USERS VIOLATING ANY OF THE POLICIES IN THIS GUIDE WILL
-> HAVE THEIR DATA STAGING ACCESS AND/OR CHTC ACCOUNT REVOKED UNTIL CORRECTIVE
-> MEASURES ARE TAKEN. CHTC STAFF RESERVE THE RIGHT TO REMOVE ANY
-> PROBLEMATIC USER DATA AT ANY TIME IN ORDER TO PRESERVE PERFORMANCE.**
+   * If your jobs need multiple large input files, 
+use `tar` and `zip` to combine files so that only a single `tar` or `zip` 
+archive is needed per job.
 
+   * For more details, see [Use The Transfer Server To Move Files To/From Staging](#transfer).
```
Comment (ChristinaLK) [inline file-avail-largedata.md:129]: shouldn't this be after the next point?

File: file-avail-largedata.md [hunk 7]
```diff
@@ -1,299 +1,603 @@
...
-SOON AS IT IS NO LONGER NEEDED FOR ACTIVELY-RUNNING JOBS. 
-- **Monitor usage and quota**: Each `/staging` folder has both a size and "items" quota. Quota changes 
-can be requested by emailing chtc@cs.wisc.edu. 
+   ```
+   tar -czf large_output.tar.gz output-file-1 output-file-2 output_dir/
+   mv large_output.tar.gz /staging/username
+   ```
+   {:.file}
+
+   * Before job completes, delete input copied from `staging`, the
```
Comment (ChristinaLK) [inline file-avail-largedata.md:167]:
```suggestion
   * Before the job completes, delete input copied from `staging`, the 
```

**Review by ChristinaLK — COMMENTED** [review]
looks good. I lost steam partway through, can try to read again before merging. :)

File: file-avail-largedata.md (hunk 2 above)
Comment (jmvera255) [reply file-avail-largedata.md:50]: I got this wording from @lmichael107

**Review by ChristinaLK — APPROVED** [review]
I think this is good to go - I just want to double check that wrapping markdown formatting in html tags will work.

**Pull request merged**